{
 "PRed": [
  {
   "PR": {
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python310"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "number": 2,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python311"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  }
 ],
 "bad": {
  "exception": "[Errno 2] No such file or directory: '/home/runner/work/cf-scripts/cf-scripts/cf-graph/enterprise_outliers-feedstock'",
  "traceback": [
   "Traceback (most recent call last):",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 567, in _run_migrator_on_feedstock_branch",
   "    migrator_uid, pr_json = run(",
   "                            ^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 211, in run",
   "    migration_run_data = run_migration(",
   "                         ^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/migration_runner.py\", line 63, in run_migration",
   "    return run_migration_containerized(",
   "           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^",
   "  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/migration_runner.py\", line 136, in run_migration_containerized",
   "    logger.debug(f\"host feedstock dir {feedstock_dir}: {os.listdir(feedstock_dir)}\")",
   "                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^",
   "FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/cf-scripts/cf-scripts/cf-graph/enterprise_outliers-feedstock'",
   ""
  ]
 },
 "pinning_version": "2022.11.06.17.50.32",
 "pre_pr_migrator_attempts": {
  "python312": 60
 },
 "pre_pr_migrator_status": {
  "python312": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9460891531'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_64_python3.12.____cpython: Cannot solve the request because of: The following packages are incompatible\n\u251C\u2500 pint-pulsar * can be installed with any of the following options:\n\u2502  \u251C\u2500 pint-pulsar 0.8 | 0.8.1 | 0.8.2 | 0.8.4 | 0.8.4 | 0.8.5 | 0.8.7 | 0.8.8 | 0.8.8 | 0.9.0 | 0.9.1 | 0.9.1 | 0.9.2 | 0.9.3 | 0.9.5 | 0.9.6 | 0.9.7 | 0.9.8 | 1.0 would require\n\u2502  \u2502  \u2514\u2500 python >=3.9,<3.10.0a0, which can be installed with any of the following options:\n\u2502  \u2502     \u2514\u2500 python 3.9.10 | 3.9.12 | 3.9.16 | 3.9.17 | 3.9.18 | 3.9.18\n\u2502  \u251C\u2500 pint-pulsar 0.6.3 | 0.7 | 0.8 | 0.8 | 0.8.1 | 0.8.2 | 0.8.4 | 0.8.4 | 0.8.5 | 0.8.7 | 0.8.8 | 0.8.8 | 0.9.0 | 0.9.1 | 0.9.1 | 0.9.2 | 0.9.3 | 0.9.5 | 0.9.6 | 0.9.7 | 0.9.8 | 1.0 would require\n\u2502  \u2502  \u2514\u2500 python >=3.8,<3.9.0a0, which can be installed with any of the following options:\n\u2502  \u2502     \u2514\u2500 python 3.8.12 | 3.8.13 | 3.8.16\n\u2502  \u251C\u2500 pint-pulsar 0.6.3 | 0.7 | 0.8 | 0.8 | 0.8.1 | 0.8.2 | 0.8.4 | 0.8.4 | 0.8.5 | 0.8.7 | 0.8.8 would require\n\u2502  \u2502  \u2514\u2500 python >=3.7,<3.8.0a0, which can be installed with any of the following options:\n\u2502  \u2502     \u2514\u2500 python 3.7.9 | 3.7.9 | 3.7.10 | 3.7.10 | 3.7.12\n\u2502  \u2514\u2500 pint-pulsar 0.6.3 | 0.7 | 0.8 | 0.8 | 0.8.1 | 0.8.2 would require\n\u2502     \u2514\u2500 python >=3.6,<3.7.0a0, which can be installed with any of the following options:\n\u2502        \u2514\u2500 python 3.6.9 | 3.6.9 | 3.6.9 | 3.6.12 | 3.6.12\n\u2514\u2500 python_abi 3.12.* *_cp312 cannot be installed because there are no viable options:\n   \u2514\u2500 python_abi 3.12 would constrain\n      \u2514\u2500 python 3.12.* *_cpython , which conflicts with any installable versions previously reported\n</pre><pre>osx_64_python3.12.____cpython: Cannot solve the request because of: The following packages are incompatible\n\u251C\u2500 pint-pulsar * can be installed with any of the following options:\n\u2502  \u251C\u2500 pint-pulsar 0.8 | 0.8.1 | 0.8.2 | 0.8.4 | 0.8.4 | 0.8.5 | 0.8.7 | 0.8.8 | 0.8.8 | 0.9.0 | 0.9.1 | 0.9.1 | 0.9.2 | 0.9.3 | 0.9.5 | 0.9.6 | 0.9.7 | 0.9.8 | 1.0 would require\n\u2502  \u2502  \u2514\u2500 python >=3.9,<3.10.0a0, which can be installed with any of the following options:\n\u2502  \u2502     \u2514\u2500 python 3.9.10 | 3.9.12 | 3.9.16 | 3.9.17 | 3.9.18 | 3.9.18\n\u2502  \u251C\u2500 pint-pulsar 0.6.3 | 0.7 | 0.8 | 0.8 | 0.8.1 | 0.8.2 | 0.8.4 | 0.8.4 | 0.8.5 | 0.8.7 | 0.8.8 | 0.8.8 | 0.9.0 | 0.9.1 | 0.9.1 | 0.9.2 | 0.9.3 | 0.9.5 | 0.9.6 | 0.9.7 | 0.9.8 | 1.0 would require\n\u2502  \u2502  \u2514\u2500 python >=3.8,<3.9.0a0, which can be installed with any of the following options:\n\u2502  \u2502     \u2514\u2500 python 3.8.12 | 3.8.13 | 3.8.16\n\u2502  \u251C\u2500 pint-pulsar 0.6.3 | 0.7 | 0.8 | 0.8 | 0.8.1 | 0.8.2 | 0.8.4 | 0.8.4 | 0.8.5 | 0.8.7 | 0.8.8 would require\n\u2502  \u2502  \u2514\u2500 python >=3.7,<3.8.0a0, which can be installed with any of the following options:\n\u2502  \u2502     \u2514\u2500 python 3.7.9 | 3.7.9 | 3.7.10 | 3.7.10 | 3.7.12\n\u2502  \u2514\u2500 pint-pulsar 0.6.3 | 0.7 | 0.8 | 0.8 | 0.8.1 | 0.8.2 would require\n\u2502     \u2514\u2500 python >=3.6,<3.7.0a0, which can be installed with any of the following options:\n\u2502        \u2514\u2500 python 3.6.9 | 3.6.9 | 3.6.9 | 3.6.12 | 3.6.12\n\u2514\u2500 python_abi 3.12.* *_cp312 cannot be installed because there are no viable options:\n   \u2514\u2500 python_abi 3.12 would constrain\n      \u2514\u2500 python 3.12.* *_cpython , which conflicts with any installable versions previously reported\n\n                </pre>\n                </div>\n                </details>"
 },
 "smithy_version": "3.21.3"
}